File: Backend/RoomMe/RoomMe.API/Models/ShoppingListGetModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RoomMe.API.Models
{
    public class ShoppingListGetModel
    {
        public int Id { get; set; }
        public int FlatId { get; set; }
        public int? CompletorId { get; set; }
        public string CompletorName { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime CompletionDate { get; set; }
        public List<ProductModel> Products { get; set; }
    }
}

File: Android-app/domain/src/main/java/uj/roomme/domain/shoppinglist/ShoppingListGetModel.kt
package uj.roomme.domain.shoppinglist

import uj.roomme.domain.product.ProductModel
import java.time.OffsetDateTime
import java.util.*

data class ShoppingListGetModel(
    val id: Int,
    val flatId: Int,
    val completorId: Int?,
    val completorName: String?,
    val name: String,
    val description: String?,
    val creationDate: OffsetDateTime,
    val completionDate: OffsetDateTime,
    val products: List<ProductModel>,
    val receipts: List<UUID>
)
